qa-covid19.planx-pla.net/etlMapping.yaml

```diff
@@ -156,6 +156,14 @@ mappings:
           - name: vaccine_community_reported_inventory
           - name: vaccine_reported_inventory
         sorted_by: updated_datetime, desc
+        flatten_props:
```

Comment:
we can remove this `flatten_props` since we have declared it previously.
and reduce the indent from line 160 to 165